tests/install/prepare.pm

```diff
@@ -5,6 +5,9 @@ use utils qw(login disable_packagekit switch_to_root_console);
 sub run {
     login;
 
+    # Temorary workaround for selinux https://progress.opensuse.org/issues/178822
```

Comment:
maybe `record_soft_failure` to really remind us that this is only temporary?

Reply:
```suggestion
    # Temporary workaround for SELinux https://progress.opensuse.org/issues/178822
```

> maybe record_soft_failure to really remind us that this is only temporary?

It's not a failure at this point... 🤔

Reply:
true, never mind then

Reply:
@nicksinger I think your suggestion is valid and it seems @kalikiana might not know what we use `record_soft_failure` for usually. Here for sure we could use `record_soft_failure` to record that reference and IMHO it would be helpful however we should then combine it with a check to see if the workaround is actually needed. Here it's applied unconditionally.

Reply:
Right. I was missing the words. The condition that will eventually make it work is what's missing here.